hands-on/launch-app.html.md.erb

```diff
@@ -10,55 +10,65 @@ redirect_from:
+
 ```output 
 ? Select organization: Personal (personal)
 ```
-**Organizations**: Organizations are a way of sharing applications and resources between Fly.io users. Every Fly.io account has a personal organization, called `personal`, which is only visible to your account. Let's select that for this guide.
+
+**Organizations**: Organizations are a way of sharing applications and resources between Fly.io users. Every Fly.io account has a personal organization, called `personal`, which is only visible to your account. Select the 'personal' organization for this example.
 
 <div class="callout">
-    If you did not add another organization to your account, the `personal` organization will be selected automatically.  
+    If you didn't add another organization to your account, the `personal` organization is selected automatically.  
 </div>
 
 Next, you'll be prompted to select a region to deploy in.
 
 ```output
-? Select region: ord (Chicago, Illinois (US))
+? Choose a region for deployment: Chicago, Illinois (US) (ord)
 ```
 
-At this point, flyctl creates a shell of an app for you and writes a starter configuration to a `fly.toml` file. 
+At this point, `flyctl` creates a shell of an app for you and writes a starter configuration to a `fly.toml` file. 
 
 <div class="callout">
 
-If you have just signed up, you will also be prompted for credit card payment information, required for charges outside the free allowances on Fly.io. See [How We Use Credit Cards](/docs/about/credit-cards/) and [Pricing](/docs/about/pricing) for more details.
+If you've just signed up, then you'll also be prompted for credit card payment information, which is required for charges outside the free allowances on Fly.io. See [How We Use Credit Cards](/docs/about/credit-cards/) and [Pricing](/docs/about/pricing) for more details.
 
 </div>
 
-The CLI will ask if you need a Postgres or Redis database. You can reply "No" for this demonstration.
+The CLI will ask if you need a Postgres or Redis database. You can enter "No" for this example.
 
 ```output
 ? Would you like to set up a Postgresql database now? No
 ? Would you like to set up an Upstash Redis database now? No
 ```
 
-Lastly, the CLI will ask you if you would like to deploy now&mdash;if you're deploying an app that doesn't use `internal_port = 8080`, please say no, and take a moment to edit `fly.toml`.
+Finally, the CLI will ask you if you would like to deploy.
+
+<div class="callout">
+If you're deploying an app that doesn't use `internal_port = 8080`, then enter "No", and edit the `fly.toml` file.
+<div>
 
 ```output
 ? Would you like to deploy now? Yes
 ==> Building image
+...
 ```
 
-Whether you deploy straight away or not, `flyctl` will download a `fly.toml` configuration file to the working directory, for use on the next deployment.
+Whether you deploy right away or not, `flyctl` will download a `fly.toml` configuration file to the working directory, for use on the next deployment.
```

Comment:
? Curious about this one

Reply:
This was a habit of using American English (and spelling). Not sure how helpful this change would be to non-English as a first language speakers though.